pkg/api/optionlevel_jsonenums.go

```diff
@@ -0,0 +1,65 @@
+// generated by jsonenums -type=OptionLevel; DO NOT EDIT
```

Comment:
Why is this required ? You have already set the field as string type in response.

Reply:
I have taken up in the middle, I have no idea why this is added, do you want me to remove it?

Reply:
Yes, as it's already a string in field of response, you can remove it.